app/components/Views/RevealPrivateCredential/RevealPrivateCredential.tsx

```diff
@@ -296,67 +299,69 @@ const RevealPrivateCredential = ({
         if (apiLevel < AppConstants.LEAST_SUPPORTED_ANDROID_API_LEVEL) {
-          setIsAndroidSupportedVersion(false);
+          setClipboardEnabled(false);
+          return;
         }
       });
 
-    return (
-      <ScrollableTabView
-        renderTabBar={() => renderTabBar()}
-        onChangeTab={(event: any) => onTabBarChange(event)}
+    setClipboardEnabled(true);
+  }, []);
+
+  const renderTabView = (privCredentialName: string) => (
+    <ScrollableTabView
+      renderTabBar={() => renderTabBar()}
+      onChangeTab={(event: any) => onTabBarChange(event)}
+    >
+      <View
+        tabLabel={strings(`reveal_credential.text`)}
+        style={styles.tabContent}
       >
-        <View
-          tabLabel={strings(`reveal_credential.text`)}
-          style={styles.tabContent}
-        >
-          <Text style={styles.boldText}>
-            {strings(`reveal_credential.${privCredentialName}`)}
-          </Text>
-          <View style={styles.seedPhraseView}>
-            <TextInput
-              value={clipboardPrivateCredential}
-              numberOfLines={3}
-              multiline
-              selectTextOnFocus
-              style={styles.seedPhrase}
-              editable={false}
-              testID={'private-credential-text'}
-              placeholderTextColor={colors.text.muted}
-              keyboardAppearance={themeAppearance}
+        <Text style={styles.boldText}>
+          {strings(`reveal_credential.${privCredentialName}`)}
+        </Text>
+        <View style={styles.seedPhraseView}>
+          <TextInput
+            value={clipboardPrivateCredential}
+            numberOfLines={3}
+            multiline
+            selectTextOnFocus
+            style={styles.seedPhrase}
+            editable={false}
+            testID={'private-credential-text'}
+            placeholderTextColor={colors.text.muted}
+            keyboardAppearance={themeAppearance}
+          />
+          {clipboardEnabled ? (
+            <Button
+              label={strings('reveal_credential.copy_to_clipboard')}
+              variant={ButtonVariants.Secondary}
+              size={ButtonSize.Sm}
+              onPress={() =>
+                copyPrivateCredentialToClipboard(privCredentialName)
+              }
+              testID={'private-credential-touchable'}
+              style={styles.clipboardButton}
             />
-            {isAndroidSupportedVersion && (
-              <TouchableOpacity
-                style={styles.privateCredentialAction}
-                onPress={() =>
-                  copyPrivateCredentialToClipboard(privCredentialName)
-                }
-                testID={'private-credential-touchable'}
-              >
-                <Text style={styles.blueText}>
-                  {strings('reveal_credential.copy_to_clipboard')}
-                </Text>
-              </TouchableOpacity>
-            )}
-          </View>
+          ) : null}
```

Comment:
Why not using && here instead of ternary with null second term?

```suggestion
{clipboardEnabled && <Button
              label={strings('reveal_credential.copy_to_clipboard')}
              variant={ButtonVariants.Secondary}
              size={ButtonSize.Sm}
              onPress={() =>
                copyPrivateCredentialToClipboard(privCredentialName)
              }
              testID={'private-credential-touchable'}
              style={styles.clipboardButton}
            />}
```

Reply:
I can't find the exact article that explained why this way was better than `&&`. @owencraston should it to me one time

Reply:
I also have similar question. Please do share with me when you do accidentally bump into it. TY